gradio/cli.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+import gradio.deploy_space
+import gradio.reload
+
+
+def cli():
+    args = sys.argv[1:]
+    if len(args) == 0:
+        raise ValueError("No file specified.")
+    if args[0] == "--deploy":
+        gradio.deploy_space.deploy()
+    else:
+        gradio.reload.run_in_reload_mode(*args)
```

Comment:
I cannot reproduce this, can you expand on this? By the way if you are in editable mode (`pip install -e .`) you may have to rerun it since I don't think CLI bindings automatically update.